README.md

```diff
@@ -22,7 +63,35 @@ Or install it yourself as:
 
 ## Usage
 
-TODO: Write usage instructions here
+To use the library create a new instance of `TableUnspanner::UnspannedTable`, passing it a `table` keyword argument that contains an HTML string representing your table. Then call the `#html_string` method which will return a string containing the processed HTML table.
```

Comment:
This still implies the only argument is `html_string`, when the example later implies that `nokogiri_node` is the primary method, with a fallback of `html_string`.